index.js

```diff
@@ -6,38 +6,41 @@ class CleanTerminalPlugin {
 
     this.message = message;
     this.onlyInWatchMode = onlyInWatchMode;
+    this.doClearConsole = false;
```

Comment:
What's the purpose of `doClearConsole`?

Reply:
Ow, that needs to go! Will amend the pull request tomorrow to fix this!

Reply:
Variable removed in latest commit